test/functional/p2p_addrv2_relay.py

```diff
@@ -18,12 +18,19 @@
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import assert_equal
 
+I2P_ADDR = "c4gfnttsuwqomiygupdqqqyy5y5emnk5c73hrfvatri67prd7vyq.b32.i2p"
+
 ADDRS = []
 for i in range(10):
     addr = CAddress()
     addr.time = int(time.time()) + i
     addr.nServices = NODE_NETWORK | NODE_WITNESS
-    addr.ip = "123.123.123.{}".format(i % 256)
+    # Add one I2P address at an arbitrary position.
+    if i == 5:
+        addr.net = addr.NET_I2P
+        addr.ip = I2P_ADDR
+    else:
+        addr.ip = "123.123.123.{}".format(i % 256)
```

Comment:
nit, f-strings for string interpolation

linter needs appeasing for "test/functional/p2p_addrv2_relay.py:19:1: F401 'test_framework.util.assert_equal' imported but unused"

Reply:
will review in any case very shortly

Reply:
Done.